### PR TITLE
Add dimmer strategy configuration for ubisys D1(-R)

### PR DESCRIPTION
**Pull request by guydriesen**

### CODE REVIEW

**Comment by guydriesen** [comment]
Relates to https://github.com/Koenkk/zigbee-herdsman/pull/277 & https://github.com/Koenkk/zigbee-herdsman-converters/pull/1984

File: docs/devices/D1.md [hunk 1]
```diff
@@ -41,6 +41,49 @@ By publishing to `zigbee2mqtt/FRIENDLY_NAME/get/ballast_config` the values of th
...
+}
+```
+
+By publishing to `zigbee2mqtt/FRIENDLY_NAME/get/ubisys_dimmer_setup` the values of the dimmer strategy configuration attributes can also be read back from the device and be printed to the normal Zigbee2MQTT log (flagged as warnings but only to make sure they do not get suppressed). More information about the meaning of the returned bitmaps can be found in the [ubisys D1 technical reference manual](https://www.smarthome-store.de/media/documents/ubisys-d1-technical-reference.pdf), chapter "7.2.8. Dimmer Setup Cluster (Server)".
+
+* Capabilities Attribute
+
+Bit(s) | Description
+---|---
+0 | When this bit is set, the dimmer supports AC forward phase control.
```
Comment (Koenkk) [inline docs/devices/D1.md:61]: Can we translate this in the converters to something more user friendly?

**Comment by Koenkk** [comment]
Can you rebase this branch against master? Most is already in there: https://github.com/Koenkk/zigbee2mqtt.io/blob/master/docs/devices/D1.md#capabilities_forward_phase_control-binary (generated from exposes info)

**Comment by guydriesen** [comment]
Didn't know that most part of the file was generated. I think it's ok now.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**